defaults/main.yml

```diff
@@ -362,7 +362,15 @@ apache__combined_snippets: '{{ apache__dependent_snippets
 apache__https_enabled: '{{ ansible_local|d() and ansible_local.pki|d() and
                            (ansible_local.pki.enabled|d() | bool) and
                            apache__https_listen|length > 0 }}'
+                                                                   # ]]]
+
 
+# .. envvar:: apache__redirect_to_https [[[
+#
+# Each vhost can define a boolean ``redirect_to_https``.
+# If not set, it defaults to the value defined in ``apache__redirect_to_https``.
+# Defaults to ``True``.
```

Comment:
Ah, yeah.  I wasn't sure how to handle that. I think I wanted to express that the ultimate decision is made based on the other variable but also make the documentation useful, e.g. to make it show the actual value that's relevant to the user.  But I don't like the redundancy.
What do you think should be done here?  Make it `Defaults to apache__redirect_to_https` and remove the previous sentence?